.mergify.yml

```diff
@@ -10,12 +10,20 @@ pull_request_rules:
   - name: automatic merge
     conditions:
       - label!=DNM
-      - '#approved-reviews-by>=2'
-      - 'status-success=continuous-integration/travis-ci/pr'
+      - "#approved-reviews-by>=2"
+      - "status-success=continuous-integration/travis-ci/pr"
     actions:
       merge:
         method: rebase
         rebase_fallback: merge
         strict: smart
       dismiss_reviews: {}
       delete_head_branch: {}
+  - name: backport patches to release branch
+    conditions:
+      - base=master
+      - label=backport-to-release-v1.2.0
```

Comment:
There should be a document with the meanings of the labels. This change should add the automatic backport feature to that doc. But I can not find the document... Maybe `./docs/development-guide.md` would be the right place?

Reply:
updated doc

Reply:
Thanks!